_includes/head.html

```diff
@@ -31,8 +31,17 @@
 
   <meta name="viewport" content="width=device-width, initial-scale=1">
   
-  <script defer src="../assets/js/fontawesome-solid.js"></script>
-  <script defer src="../assets/js/fontawesome-main.js"></script>
+    
+  {% if page.parent %}
+    <script defer src="../../assets/js/fontawesome-solid.js"></script>
+    <script defer src="../../assets/js/fontawesome-main.js"></script>
+  {% else if parent %} 
```

Comment:
What does parent refer to ? All page attributes are available via **page.something** syntax. Might consider removing this.